src/index.ts

```diff
@@ -362,8 +362,8 @@ export function initializePAL(callback: (platform: Platform, feature: Feature, d
     return;
   }
   isInitialized = true;
-  if (typeof Object.getPropertyDescriptor !== 'function') {
-    Object.getPropertyDescriptor = function(subject, name) {
+  if (typeof (Object as any).getPropertyDescriptor !== 'function') {
```

Comment:
Shouldn't we extend the interface of Object here ? Or is this because we just want it to be used internally?

Reply:
If it's meant to be public, yes absolutely.
And you're spot on, this particular line is something I'd like to discuss.

I'll open a PR with code changes at a later point. I wanted to get the "infrastructure" of the migration first.